numpy/f2py/_backends/_meson.py

```diff
@@ -126,7 +126,7 @@ def generate_meson_build(self):
             node()
         template = Template(self.meson_build_template())
         meson_build = template.substitute(self.substitutions)
-        meson_build = re.sub(r",,", ",", meson_build)
+        meson_build = meson_build.replace(",,", ",")
```

Comment:
`re` would probably be faster, but I doubt that the difference would really matter here

Reply:
Are you certain that `re.sub` is faster than `str.replace`, especially if you take into account overhead of (not-)compiling the (not-)regex?

Reply:
but I'm guessing that it'll depend on the length `meson_build`

Reply:
[10 Popular String Operations in Python: Performance Comparison](https://getanteon.com/blog/10-Popular-String-Operations-in-Python/)

Reply:
I doubt the speed of this operation is noticeable in the overall build time.